CHANGELOG.rst

```diff
@@ -1,5 +1,42 @@
 Changelog
 =========
+1.19.1.dev0 (2021-08-25)
+------------------------
+MLflow 1.19.1.dev0 includes several major features and improvements:
+
+Features:
+
+- MLflow UI updates (#4712, @dbczumar)
+- Support distributed pytorch lightning autologging execution (#4717, @dbczumar)
+- Added more Model Registry Functions to R (#4527, @bramrodenburg)
+- Pyspark ml autolog post-training metric (#4686, @WeichenXu123)
+- Scoring Server - Added support for wrapped prediction responses (#4611, @Ark-kun)
+- make sklearn autologging supporting delegated methods in estimators (#4628, @WeichenXu123)
+- Pyspark CrossValidator autolog "std metrics" (#4672, @WeichenXu123)
+- Implement auto pip dependency inference for `mlflow.*.log_model` (#4518, @harupy)
+- sklearn eval metric autologging: patch `metrics.SCORERS` (#4638, @WeichenXu123)
+- Added archive argument to the Sagemaker Batch Transform integration (#4589, @YQ-Wang)
+- Deprecate  argument for pytorch flavor (#4620, @harupy)
+- Autolog post training metrics for sklearn (#4491, @WeichenXu123)
+- [PART1] Add `pip_requirements` and `extra_pip_requirements` to all flavors except xgboost flavor (#4577, @harupy)
+- [PART 2] Add `pip_requirements` and `extra_pip_requirements` to all flavors except xgboost flavor (#4602, @harupy)
+- Implemented Sagemaker Batch Transform integration with MLflow (#4410, @YQ-Wang)
+- [ML-16200] Add `pip_requirements` and `additional_pip_requirements` to `log_model` (#4519, @harupy)
```

Comment:
Entries for `pip_requirements` and `extra_pip_requirements` can be merged into one entry.

Reply:
@harupy Can you write the unified release entry for these new arguments and push it to this PR?

Reply:
Absolutely, what about:

```
Add `pip_requirements` and `extra_pip_requirements` to `mlflow.*.save_model` and `mlflow.*.log_model` for directly specifying pip requirements of the model?
```

Reply:
Perfect!